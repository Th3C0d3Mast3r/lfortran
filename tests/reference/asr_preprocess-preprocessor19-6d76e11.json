{
    "basename": "asr_preprocess-preprocessor19-6d76e11",
    "cmd": "lfortran --cpp --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/preprocessor19.f90",
    "infile_hash": "8a52da8afe0f8e0b7710633bf53282015e2a18b69ad344b5e4580c85",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr_preprocess-preprocessor19-6d76e11.stderr",
    "stderr_hash": "b0c670d1966e0be866b7669dc77d58ff610b89cdf474b234d9bbd5ca",
    "returncode": 2
}