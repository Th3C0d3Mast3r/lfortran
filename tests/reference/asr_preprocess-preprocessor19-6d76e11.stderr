warning: #define ignored
 --> tests/preprocessor19.f90:1:1
  |
1 | #define Fn(x)
  | ^^^^^^^^^^^^^^ help: use the '--cpp' command line option to preprocess it

syntax error: Newline is unexpected here
 --> tests/preprocessor19.f90:5:8
  |
5 |   Fn(1)
  |        ^ 
